CONTRIBUTING.md

```diff
@@ -21,6 +20,9 @@ Otherwise, when filing an issue, make sure to answer these five questions:
 4. What did you expect to see?
 5. What did you see instead?
 
+Note:
+When you'd like to contribute with the new feature of some large update influencing the current Vald, we recommend getting the agreement through the authors' design review of your ISSUE.
+
 ## Before contributing code
```

Comment:
suggest adding a link to the coding guideline in this section

Reply:
I think we should add after #320 is merged, how do you think?

Reply:
you are right, then I will include this changes in #320

Reply:
thanks !